compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -347,6 +349,14 @@ public BType getLangLibSubType(String name) {
         throw new IllegalStateException("LangLib Subtype not found: " + name);
     }
 
+    public void loadPredeclaredModules() {
+        this.predeclaredModules.put(Names.ERROR, this.langErrorModuleSymbol);
+        this.predeclaredModules.put(Names.OBJECT, this.langObjectModuleSymbol);
+        this.predeclaredModules.put(Names.XML, this.langXmlModuleSymbol);
+
+        this.predeclaredModules = Collections.unmodifiableMap(this.predeclaredModules);
```

Comment:
What I meant was something like
```java
Map<Name, BPackageSymbol> modules = new HashMap<>();
modules.put(Names.ERROR, this.langErrorModuleSymbol);
modules.put(Names.OBJECT, this.langObjectModuleSymbol);
modules.put(Names.XML, this.langXmlModuleSymbol);

this.predeclaredModules = Collections.unmodifiableMap(modules);
```

Reply:
It gives NullPointerException while building blang packages in this way.

Reply:
Changed (As mention in the https://github.com/ballerina-platform/ballerina-lang/pull/23884#discussion_r440125692)